scripts/docker/init.sql

```diff
@@ -0,0 +1,4 @@
+create role nmdc_data_reader;
+
+create database nmdc_a;
+create database nmdc_b;
```

Comment:
FYI I did need to do a `docker compose down -v` in order for this script to run. Logs suggest that this script is skipped if there is any DB already present. Might be worth mentioning in the docs that existing users will see an error if the `nmdc_data_reader` role doesn't exist locally for them.